templates/terraform/.envrc

```diff
@@ -2,9 +2,8 @@ if ! has nix_direnv_version || ! nix_direnv_version 2.2.1; then
   source_url "https://raw.githubusercontent.com/nix-community/nix-direnv/2.2.1/direnvrc" "sha256-zelF0vLbEl5uaqrfIzbgNzJWGmLzCmYAkInj/LNxvKs="
 fi
 
-nix_direnv_watch_file devenv.nix
-nix_direnv_watch_file devenv.lock
-nix_direnv_watch_file devenv.yaml
+nix_direnv_watch_file flake.nix
+nix_direnv_watch_file flake.lock
```

Comment:
Updating this file based on the [simple](https://github.com/cachix/devenv/blob/main/templates/simple/.envrc) template.